src/app/Library/CrudPanel/Traits/Read.php

```diff
@@ -40,8 +40,10 @@ public function getCurrentEntry()
     {
         $id = $this->getCurrentEntryId();
 
-        if (! $id) {
-            return false;
+        if (is_bool($id)) {
+            if (! $id) {
+                return false;
+            }
         }
```

Comment:
```suggestion
        if ($id === false) {
                return false;
        }
```

I'm wondering - would this do the same thing @valebar ? It looks like it to me, and it'd be a little cleaner right?

Reply:
sorry... I accidently pushed the code you suggested :) mea culpa

Reply:
No problem at all - so did you get to test it, works well in your situation?

Reply:
it works as expected.